common/src/main/scala/net/psforever/packet/game/SquadState.scala

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) 2019 PSForever
+package net.psforever.packet.game
+
+import net.psforever.packet.{GamePacketOpcode, Marshallable, PlanetSideGamePacket}
+import net.psforever.types.Vector3
+import scodec.{Attempt, Codec, Err}
+import scodec.codecs._
+import shapeless.{::, HNil}
+
+/**
+  * Information about a specific squad member.
+  * @param char_id the character's unique identifier
+  * @param health the character's health value percentage, divided into 64 units
+  * @param armor the character's armor value percentage, divided into 64 units
+  * @param pos the world coordinates of the character
+  * @param unk4 na;
+  *             usually, 2
+  * @param unk5 na;
+  *             usually, 2
```

Comment:
It's hard to determine if a packet field being constant warrants decoding it as a full int or as a fixed value. One leads to easier debugging when creating the packet, but moves the burden of providing the bespoke fixed values on the packet creator. Not sure if this is reasonable.

Reply:
If it becomes a packet I revisit a lot in code, I just write a constructor that inserts those values in between the other populated fields.